WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -191,7 +193,17 @@ extension ReaderTabViewController {
     private enum ReaderTabConstants {
         static let title = NSLocalizedString("Reader", comment: "The default title of the Reader")
         static let settingsButtonIdentifier = "ReaderSettingsButton"
+        static let settingsButtonAccessibilityLabel = NSLocalizedString(
+            "reader.navigation.settings.button.label",
+            value: "Reader Settings",
```

Comment:
I was thinking of ditching the "Reader" part from "Reader Settings" to align it with the "Search" accessibility label. This is just a nit, I'll let you decide whether to include this change or not.

```suggestion
            value: "Settings",
```

Reply:
Haha, ChatGPT pointed out the same thing. I thought "Settings" might be not very obvious as it could mean settings of a lot of things whereas "Search" felt less ambiguous. I figured making it a little more obvious might help to a visually impaired person.